sources/packages/api/src/database/entities/institution-user-type-role.model.ts

```diff
@@ -0,0 +1,28 @@
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+import { RecordDataModel } from "./record.model";
+
+@Entity({
+  name: "institution_user_type_roles",
+})
+export class InstitutionUserTypeAndRole extends RecordDataModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({
+    name: "user_type",
+    nullable: false,
+  })
+  type: string;
+
+  @Column({
+    name: "user_role",
+    nullable: true,
+  })
+  role: string;
```

Comment:
Can we create the string enums with the defined roles?

Reply:
The enums are created but not attached with type def, I will do that in upcoming PR